tests/post/steps/test_monitoring.py

```diff
@@ -174,6 +182,69 @@ def _node_has_metrics():
     utils.retry(_node_has_metrics, times=60, wait=3)
 
 
+@then(
+    "the deployed Prometheus alerting rules is the same as the default "
+    "alerting rules")
+def monitor_deployed_rules(host):
```

Comment:
Same comment, this is not about monitoring rules...

```suggestion
def check_deployed_rules(host):
```